backend/domain/Room.js

```diff
@@ -39,13 +38,32 @@ class Room {
     }
 
     getPlayerByNickname(nickname) {
-        for (player of this.players) {
```

Comment:
this was just missing a const in for(const player of this.players). New changes are fine too.